mod-source-record-manager-server/pom.xml

```diff
@@ -232,6 +232,17 @@
       <artifactId>util</artifactId>
       <version>33.0.3</version>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <version>3.20.2</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
+      <version>2.6</version>
+    </dependency>
```

Comment:
I'm pretty sure that this dependency already exists in the classpath.

Reply:
![image](https://user-images.githubusercontent.com/88025830/130409730-cb24f2e0-de6f-4b83-9d12-c0380fd48de4.png)
It's not my changes...